### ISSUE TITLE
[Site] Footer improvements suggestions

### ISSUE THREAD

**Issue by helenea** [openmole/openmole#359]
Little things that could be improved in the future:

- the side menus are hidden by the footer at the bottom of the page

- the content of the footer is not centered with the content of the main text

- the space before the footer content could be reduced a bit

![image](https://user-images.githubusercontent.com/8794131/43148444-8c950d6a-8f65-11e8-880d-73286fb9973c.png)

**Comment by mathieuleclaire** [comment]
What do you think of e6fefb2c965f0a17d5abbd42249946e75b79ec96 ? It should be better. It enables to have the whole menus on 13' screen. Then, if we want to deal with smaller screens. 

Other ideas:
- limit the number of anchors per page (not my best idea at all) or,
- present Available Tasks/Methods/Scale on two columns 
- or present only main Tasks/Methods/Scale in the side menus and add a "Other" button linked to the main Tasks/Methods/Sacle page.

**Comment by helenea** [comment]
I like the reduced padding at the top of the footer, however it's not centered and I still don't have the whole menus on my 14" screen :/

The other ideas I have have more to do with the side menus than with the footer itself but here it goes:
- I think having anchors and the sections menu both on the left-hand side is a bit much. Maybe the anchors don't need to be visible during scrolling and could only be at the top of the page serving as a table of content for long pages.
- Maybe a different type of buttons taking less space, or just links
- Making the menus stick to the top of the page when scrolling would give a bit more space but that's just going around the problem

**Comment by mathieuleclaire** [comment]
It is weird, because the longest menu is displayed correctly on my 13,3'' ... do you set a big font size ?
![capture du 2018-08-02 15-09-01](https://user-images.githubusercontent.com/800035/43585826-56f86d16-9666-11e8-9ab8-8abdb76e2c9f.png)

**Comment by mathieuleclaire** [comment]
1098833cd8dd769e5472e6d100e8f6392ef2969a fixes footer alignement.

**Comment by mathieuleclaire** [comment]
OK; so it's a screen resolution issue. I am not sure that hidding the menu when scrolling down is a good behaviour. Peharps should we list the main Tasks (with a "other" button pointing to the Documentation page) and let the other accessible only from the Documentation page.

**Comment by helenea** [comment]
I didn't mean hiding the anchors but rather put them at the top of the main text as toc instead of in the left menu, Wikipedia style (but prettier).
I like the idea of an "other" button too!

**Comment by mathieuleclaire** [comment]
0c73ad5d7b57d262d69949b670d4c3e05a678535 put them at the top. Better ?

**Comment by helenea** [comment]
Better but still not enough to get the whole menu, on my screen at least :/

**Comment by helenea** [comment]
You don't like the idea of putting the anchors in the main text? They would not be visible at all times but it would leave more space for the menu.

**Comment by mathieuleclaire** [comment]
Like this ? The problem is we can only use them from the top page. But is it really a problem ?
![capture du 2018-08-07 18-28-11](https://user-images.githubusercontent.com/800035/43790347-a67e918e-9a72-11e8-8450-9b1653db9955.png)

**Comment by helenea** [comment]
Yes exactly like this :) I don't think it's a problem but that's just me.
